Automattic/wp-calypso
# Update/mailbox link professional email without feature flags

#### Changes proposed in this Pull Request

Adds redirect url and email to the Professional Email embedded inbox. We are also removing TEMPORARILY two menu items, Calendar & Contact
![image](https://user-images.githubusercontent.com/5689927/175997945-0c457ae7-aa0c-423d-ac78-162a0c20c34f.png)
 

#### Testing instructions

1. Have a Professional Email account
2. Access it through the Inbox menu in the calypso sidebar
3. It should prompt you to  use your password (or log in if you are already authenticated)
4. The Look & Feel should have a black top bar with the WordPress logo (Similar to our dashboard)
